### PR TITLE
[0437] Correcting dialogue of creature 3520

### LINKED ISSUE #325: Old Emma Stormwind Dialogue
Ol` Emma says "ifn it werent fer me" and I think it should be "if it werent fer me"
untitled

### PR DESCRIPTION
**Pull request by Sinoxan**
- Correcting dialogue of creature 3520 "Ol' Emma"
  Thanks Swater96 for pointing. This closes #325
  Source: http://wowpedia.org/Ol%27_Emma
  Source: http://www.wowhead.com/npc=3250#comments

### CODE REVIEW

**Comment by Swater96** [comment]
Awesome thank you!

File: updates/0436_db_script_string.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Fixing dialogue of creature 3520 (Ol' Emma)
+-- Thanks Swater96 for reporting, fixes #325
+-- Source: http://wowpedia.org/Ol%27_Emma
+-- Source: http://www.wowhead.com/npc=3520#comments
+UPDATE `db_script_string` SET `content_default` = 'Jack and Jill my wrinkled patoot! I do all the water luggin\' \'round here.' WHERE `entry` = 2000005103;
+UPDATE `db_script_string` SET `content_default` = 'Where\'s the water, Emma? Get the water, Emma? If\'n it werent fer me, that lot wouldn\'t know what water looks like.' WHERE `entry` = 2000005104;
```
Comment (PatSmuk) [inline updates/0436_db_script_string.sql:6]: werent -> weren't (?)

Comment (Sinoxan) [inline updates/0436_db_script_string.sql:6]: Oops, didn't notice that one, thanks

**Comment by PatSmuk** [comment]
Accepted. Thank you! :)

**Pull request closed without merging**